src/bflib_netsession.c

```diff
@@ -30,8 +30,7 @@ void net_copy_name_string(char *dst,const char *src,long max_len)
   {
     if (src != NULL)
     {
-      strncpy(dst, src, max_len-1);
-      dst[max_len-1] = '\0';
+        snprintf(dst, max_len-1, "%s", src);
```

Comment:
-1 is not required as long as `dst` is `max_len` size. Prof: https://cplusplus.com/reference/cstdio/snprintf/